lib/commands/slicemachine/commands/init.js

```diff
@@ -201,52 +199,31 @@ async function init() {
       const [protocol, base] = ctx.base.split('://');
       if (!protocol || !base) return [`Base url "${ctx.base}" is invalid: did you forget to specify protocol?`, null];
 
-      const apiEndpointUrl = `${protocol}://${r.domain}.${ctx.base.replace(/^(?:https?:\/\/)?(?:www\.)?/i, '')}/api/v2`;
-      return [null, _format(r.domain, r.magicUrl, apiEndpointUrl)];
+      const apiEndpoint = `${protocol}://${r.domain}.${ctx.base.replace(/^(?:https?:\/\/)?(?:www\.)?/i, '')}/api/v2`;
+      return [null, _format(r.domain, r.magicUrl, apiEndpoint)];
     }
     return [null, _format()];
   })();
 
   if (error) return consola.error(error);
 
+  const configFile = Mustache.render(
```

Comment:
why do we do this here? will we have the prismic.config.js in every technology or is it nuxt related?